packer/files/rule_config.yml

```diff
@@ -0,0 +1,96 @@
+global:
+  # The smarthost and SMTP sender used for mail notifications.
+  smtp_smarthost: 'localhost:25'
+  smtp_from: 'alertmanager@example.org'
+
+# The root route on which each incoming alert enters.
+route:
+  # The root route must not have any matchers as it is the entry point for
+  # all alerts. It needs to have a receiver configured so alerts that do not
+  # match any of the sub-routes are sent to someone.
+  receiver: 'team-X-mails-regular'
+
+  # The labels by which incoming alerts are grouped together. For example,
+  # multiple alerts coming in for cluster=A and alertname=LatencyHigh would
+  # be batched into a single group.
+  # group_by: ['alertname', 'cluster']
+
+  # When a new group of alerts is created by an incoming alert, wait at
+  # least 'group_wait' to send the initial notification.
+  # This way ensures that you get multiple alerts for the same group that start
+  # firing shortly after another are batched together on the first
+  # notification.
+  group_wait: 30s
+
+  # When the first notification was sent, wait 'group_interval' to send a batch
+  # of new alerts that started firing for that group.
+  group_interval: 5m
+
+  # If an alert has successfully been sent, wait 'repeat_interval' to
+  # resend them.
+  repeat_interval: 6h
+
+  # All the above attributes are inherited by all child routes and can 
+  # overwritten on each.
+
+  # The child route trees.
+  routes:
+  - match:
+      severity: '4'
+    receiver: team-X-mails-urgent
+  - match:
+      severity: '3'
+    receiver: team-X-mails-urgent
+
+
+
+# Inhibition rules allow to mute a set of alerts given that another alert is
+# firing.
+# We use this to mute any warning-level notifications if the same alert is
+# already critical.
+inhibit_rules:
+- source_match:
+    severity: '4'
+  target_match:
+    severity: '3'
+  # Apply inhibition if the alertname is the same.
+  equal: ['alertname']
+- source_match:
+    severity: '3'
+  target_match:
+    severity: '2'
+  # Apply inhibition if the alertname is the same.
+  equal: ['alertname']
+- source_match:
+    severity: '2'
+  target_match:
+    severity: '1'
+  # Apply inhibition if the alertname is the same.
+  equal: ['alertname']
+
+
+receivers:
+- name: 'team-X-mails-regular'
+  email_configs:
+  - from: SENDER_EMAIL
+    to: TARGET_EMAIL_DESTINATION
+    smarthost: SENDER_SMTP_HOST
```

Comment:
@benipeled

who is supposed to fill those correctly ? 

and why are the image override this file, with this broken version of the configuration  ?

i.e. I'm trying to use the latest image, and I altermanager fails cause of this file

Reply:
Siren does, there's an open issue for them to fix that. until they do, I keep a copy of the original file, so it will be possible to override it back and start the stack (this is what I do when I test)